.github/update.py

```diff
@@ -8,62 +8,61 @@
 import html
 from xml.sax import saxutils as su
 
-ET.register_namespace('', "urn:oasis:names:tc:xliff:document:1.2")
-ET.register_namespace('qt', "urn:trolltech:names:ts:document:1.0")
+ET.register_namespace('', 'urn:oasis:names:tc:xliff:document:1.2')
+ET.register_namespace('qt', 'urn:trolltech:names:ts:document:1.0')
 
-VPN_PROJECT_DIR ="vpn/" 
-OUT_PROJECT_DIR ="translationFiles/"
+VPN_PROJECT_DIR = 'vpn'
+OUT_PROJECT_DIR = 'translationFiles'
 
-#LCONVERT = "/usr/lib/qt5/bin/lconvert"
-
-LCONVERT = "lconvert"
+#LCONVERT = '/usr/lib/qt5/bin/lconvert'
+LCONVERT = 'lconvert'
 
 # Make sure the Target ts files are up to date
-os.system(f'lupdate {VPN_PROJECT_DIR}src/src.pro -ts')
+srcFile = os.path.join(VPN_PROJECT_DIR, 'src', 'src.pro')
+os.system(f'lupdate {srcFile} -ts')
 
-for fileName in os.listdir(f'{VPN_PROJECT_DIR}/translations/'):
-    if(not fileName.endswith(".ts")):
+for fileName in os.listdir(os.path.join(VPN_PROJECT_DIR, 'translations')):
+    if (not fileName.endswith('.ts')):
         continue
-    filePath = f'{VPN_PROJECT_DIR}/translations/{fileName}'
-    # Usual filename -> mozillavpn_zh-cn.ts
-    language = fileName.split("_")[1].split(".")[0] # de , zh-cn ...
-    basename = fileName.split("_")[0] # mozillavpn
-    outPath = f'{OUT_PROJECT_DIR}/{language}'
-    # Create folder for each language and convert 
-    # ts file to /{lang}/mozillavpn.xlf
-    print(f"Checking {language}")
+    filePath = os.path.join(VPN_PROJECT_DIR, 'translations', fileName)
+    # Usual filename: mozillavpn_zh-cn.ts
+    locale = fileName.split('_')[1].split('.')[0] # de, zh-cn, etc.
```

Comment:
Note: the folder created by Pontoon would be `zh-CN`, not `zh-cn`. What would be the name of the file in the code's repository?

Reply:
in case the folder is `zh-CN` the .ts file would be `mozillavpn_zh-CN.ts`